mindjunkies/forums/tests/test_views.py

```
@@ -0,0 +1,369 @@
+        
+    def test_like_comment(self):
+        """Test liking a comment directly"""
+        # Directly test the like functionality
+        self.comment.likes.add(self.user)
+        self.assertTrue(self.comment.likes.filter(username=self.user.username).exists())
+        
+    def test_like_reply(self):
+        """Test liking a reply directly"""
+        # Directly test the like functionality
+        self.reply.likes.add(self.user)
+        self.assertTrue(self.reply.likes.filter(username=self.user.username).exists())
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)